prow/scripts/cluster-integration/kyma-gke-long-lasting.sh

```diff
@@ -197,11 +196,27 @@ function generateAndExportLetsEncryptCert() {
         --server https://acme-v02.api.letsencrypt.org/directory \
         --dns-google-propagation-seconds=600 \
         -d "*.${DOMAIN}"
-
+	
     TLS_CERT=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/fullchain.pem | tr -d '\n')
     export TLS_CERT
     TLS_KEY=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/privkey.pem   | tr -d '\n')
     export TLS_KEY
+	#encrypt the tls cert
+	gcloud kms encrypt --location global \
+	--keyring kyma-prow \
+	--key projects/kyma-project/locations/global/keyRings/kyma-prow/cryptoKeys/kyma-prow-encryption\
+	--plaintext-file ./letsencrypt/live/"${DOMAIN}"/fullchain.pem  \
+	--ciphertext-file "nightly-aks-tls-integration-app-client-cert.encrypted"
+	#encrypt the tls private key
+	gcloud kms encrypt --location global \
+	--keyring kyma-prow \
+	--key projects/kyma-project/locations/global/keyRings/kyma-prow/cryptoKeys/kyma-prow-encryption\
+	--plaintext-file ./letsencrypt/live/"${DOMAIN}"/privkey.pem  \
+	--ciphertext-file "nightly-aks-tls-integration-app-client-tls.encrypted"
+
+	#copy the privite key
+	gsutil cp nightly-aks-tls-integration-app-client-secret.encrypted gs://kyma-prow-secrets/
+	go run "${KYMA_PROJECT_DIR}/test-infra/development/tools/cmd/enablegithubauth/main.go"
```

Comment:
probably not needed line

Reply:
Right the cert should be good, you made me think, we will need to put the password for the cert as a environment variable no?